apps/app_voicemail.c

```diff
@@ -6184,7 +6184,6 @@ static int msg_create_from_file(struct ast_vm_recording_data *recdata)
 	#ifdef IMAP_STORAGE
 	struct vm_state *vms = NULL;
 	char ext_context[256] = "";
-	char *fmt = ast_strdupa(recdata->recording_ext);
```

Comment:
This should be a separate PR since it's to fix IMAP building

Reply:
moved to it's own pr/issue